x/poe/contract/tgrade_valset.go

```diff
@@ -75,7 +75,13 @@ type TG4ValsetExecute struct {
 	// only if the jail period passed.
 	Unjail *UnjailMsg `json:"unjail,omitempty"`
 	// UpdateAdmin set a new admin address
-	UpdateAdmin *TG4UpdateAdminMsg `json:"update_admin,omitempty"`
+	UpdateAdmin  *TG4UpdateAdminMsg `json:"update_admin,omitempty"`
```

Comment:
Nit: you are enforcing the indentation here, but not above.

Reply:
`go fmt` does that (and yeah, I find it annoying that it resets when there is a comment)